### PR TITLE
Improve website

### PR DESCRIPTION
**Pull request by wellsilver**
7 month old website needs to be improved. Ive done lots of itty improvements and will improve it alot more with suggestions and stuff, this is my first pull request

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -8,9 +7,14 @@
 -->
 
 <!--Let's get meta bois-->
+<!-- Lots of kids play FNF, keep everything visible child friendly. -->
```
Comment (ActualMandM) [inline docs/index.html:10]: very contradictory to what the game actually has

File: docs/index.html [hunk 2]
```diff
@@ -8,9 +7,14 @@
...
 <head>
 	<meta charset="utf-8">
-	<title>Friday Night Funkin'</title>
+	<title>Friday Night Funkin</title>
```
Comment (ActualMandM) [inline docs/index.html:13]: the `'` is part of the game title

**Review by ActualMandM — CHANGES_REQUESTED** [review]
[ninjamuffin99 has already declined a website PR because he wants to teach himself web design](https://github.com/ninjamuffin99/Funkin/pull/269#issuecomment-766184824)

File: docs/index.html (hunk 2 above)
Comment (VMGuy23) [reply docs/index.html:13]: yeah
```suggestion
	<title>Friday Night Funkin'</title>
```

**Comment by wellsilver** [comment]
> [ninjamuffin99 has already declined a website PR because he wants to teach himself web design](https://github.com/ninjamuffin99/Funkin/pull/269#issuecomment-766184824)

Now closed

File: docs/index.html (hunk 1 above)
Comment (upintheairsheep) [reply docs/index.html:10]: the whole point of the game is to be a love letter to the mature site Newgrounds.

Comment (wellsilver) [reply docs/index.html:10]: > the whole point of the game is to be a love letter to the mature site Newgrounds.

cool but this was a year ago

**Pull request closed without merging**